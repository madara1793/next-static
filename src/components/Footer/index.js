// @flow

import React from 'react';

export default () =>
  <footer className="footer">
    <small className="footer--madeWith">
      Made with <span role="img" aria-label="love">❤️</span> and{' '}
      <a href="https://github.com/infiniteluke/next-static">
        Next Static <span role="img" aria-label="with extra sparkles">✨</span>
      </a>
    </small>
  </footer>;
