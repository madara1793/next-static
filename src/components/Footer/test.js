import React from 'react';
import { renderToStaticMarkup as render } from 'react-dom/server';
import Footer from './index';
import renderer from 'react-test-renderer';

describe('Snapshot::<Footer /> component', () => {
  it('should render the contents of the component.', () => {
    const el = <Footer />;
    const tree = renderer.create(el).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
