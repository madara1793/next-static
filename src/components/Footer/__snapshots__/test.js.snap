// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot::<Footer /> component should render the contents of the component. 1`] = `
<header
  className="footer Footer__Footer-s9jqh5c-0 iaXmbF"
>
  <small
    className="footer--madeWith"
  >
    Made with
     
    <span
      aria-label="love"
      role="img"
    >
      ❤️
    </span>
     
    and
     
    <a
      href="https://github.com/infiniteluke/next-static"
    >
      Next Static
       
      <span
        aria-label="with extra sparkles"
        role="img"
      >
        ✨
      </span>
    </a>
  </small>
</header>
`;
